### PR TITLE
Readme. @jmburges @AnnJohn

### PR DESCRIPTION
**Pull request by queerviolet**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,125 @@
+# Authentication with Devise
+
+## Learning Objectives
+
+  1. Describe the major architecture and modules of Devise.
+
+## Overview
+
+[Devise] is a gem for when you have a lot of needs.
```
Comment (adamjonas) [inline README.md]: Maybe "authentication needs?" I know it's in the the title but on first pass it felt like there was a word missing.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,125 @@
...
+
+Want to send confirmation emails, lock user accounts after a certain number of failed login attempts, and send password resets? Devise can do that.
+
+Want to allow multiple models to be signed in, each with different roles, granting different permissions? Devise can handle that too.
+
+Devise has an extensive collection of generators. It includes standard views and controllers. It will generate templates that include Bootstrap for you.
```
Comment (adamjonas) [inline README.md:16]: Great examples.

File: README.md [hunk 3]
```diff
@@ -0,0 +1,125 @@
...
+
+It will also be, at times, a giant pain, because no magic is without a price.
```
Comment (adamjonas) [inline README.md]: Nice. For me what makes devise tough is that is abstracts so much away. Might be worth a mention.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,127 @@
...
+
+
+### [omniauthable]
+
+Honestly, this one doesn't give you a whole lot more than omniauth already does. It does set some (but not all!) of the routes for you. That's a nice touch.
+
+
+## Take a breath
+
+Devise is rather a lot to take in all at once. The thing about large and magical frameworks like Devise (and Rails for that matter) is that they aren't hard in the way that math problems are hard, but they are hard in the way that
```
Comment (adamjonas) [inline README.md]: I'm on the edge of my seat! How is it hard?? Missing a few words.

**Comment by adamjonas** [comment]
Loved the tone on this one.

File: README.md (hunk 2 above)
Comment (queerviolet) [inline README.md:16]: Thx!

**Pull request merged**